src/environments/environment.prod.ts

```diff
@@ -1,5 +1,6 @@
 export const environment = {
   production: true,
   apiUrl: 'https://xby2-website-api.azurewebsites.net',
-  googleAnalyticsTrackingId: 'UA-114675726-2'
+  googleAnalyticsTrackingId: 'UA-114675726-2',
+  baseCmsUrl: 'http://localhost/xby2/wp-json/xby2/v1'
```

Comment:
Can this be changed to use the following:

'https://api.xby2.com/wp-json/xby2/v1'